src/utils/cmds/cmds_test.c

```diff
@@ -204,6 +204,18 @@ static struct {
         CMD_OK,
         CMD_PUTVAL,
     },
+    {
+        "PUTVAL myhost/magic/MAGIC meta:KEY=\"string_value\" 1234:42",
```

Comment:
This test is invalid.
In this case `set_option()` receives `string_value` in `value`  variable.
`is_quoted()` returns false, no metadata added.